docs/eventing/sources/apiserversource/getting-started/_index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Getting started"
+weight: 01
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - /docs/eventing/samples/kubernetes-event-source
+---
+
+## Prerequisites
+
+Before you can create an API server source, you must install Knative Eventing and the `kubectl` CLI tool.
+
+## Create an API server source
+
+1. Optional: Create a namespace for the API server source instance:
+
+    ```shell
+    kubectl create namespace <new_namespace_name>
+    ```
+
+    Creating a namespace for your API server source and related components allows you to view changes and events for this workflow more easily, since these are isolated from the many other components that may exist in your `default` namespace.
+
+    It also makes removing the source easier, since you can simply delete the namespace to remove all of the resources.
+
+1. Create a service account:
+
+    ```yaml
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: <service_account_name>
```

Comment:
I am not a big fan of the `<service_account_name>` usage, it means the person following the demo can't copy and paste without edits. 

I would recommend us either:

1. Define a block of exports that will be used in the demo, like:

```shell
export service_account_name=example-serviceaccountname
```

then used 

```shell
      name: ${service_account_name}
```

Or 2. 

we just make up a demoish name that does not conflict with our other demos.

```yaml
      name: apiserversource-demo
```

Reply:
I guess it's just difference of opinion, but I feel like people should be using their own values and docs as a reference rather than just copy and pasting a whole dummy environment. 

@omerbensaadon do you have any UX research about this either way? Maybe people do just want to copy/paste commands but to me that just doesn't add any value as opposed to explaining what things actually are (i.e. teaching them to fish). Also, having callouts maybe would solve this problem because we could use the demo values but then still give an explanation... sort of why I was asking about that in Slack FYI 🙂

Reply:
@abrennan89 having callouts would totally solve this!

I don't have feedback specific to this from user interviews, but maybe there's a happy medium between these two ideas? 

Here's what Samia + Richard and I did on the last project we worked on: 
<img width="656" alt="Screen Shot 2021-03-31 at 9 12 23 AM" src="https://user-images.githubusercontent.com/16281246/113149616-38c52980-9201-11eb-96ef-a4fde6420bab.png">


1. The code block 
2. Where: and then a short description of each field? 

Maybe the way Scott has structured it (`${service_account_name}`) would make it obvious to folks that they've got to throw their own values in? 

@snneji and @RichardJJG may have opinions here though, I'm confident they've dealt with this sort of thing before...

Reply:
@abrennan89 let me know if you like this idea, I am happy to take a crack at implementing :)

Reply:
Unless I'm mistaken, I think this is already in Google's style guide. Google's recommendation says placeholder variables should be in the format `SERVICE_ACCOUNT_NAME`. See [Formatting placeholders](https://developers.google.com/style/placeholders). Callouts are also mentioned at the end of that topic.

Reply:
@omerbensaadon this is different to how Kubernetes does variables - I prefer the k8s way over the Google style guide, but I think these sorts of clashes are why we should have a Knative specific style guide that cements which ones we're using between k8s and Google styles.
https://kubernetes.io/docs/contribute/style/style-guide/#use-angle-brackets-for-placeholders